api/src/test/resources/io/strimzi/api/kafka/model/Kafka.out.yaml

```diff
@@ -25,6 +23,8 @@ spec:
         - podSelector:
             matchLabels:
               app: "kafka-producer"
+        authentication:
+          type: "scram-sha-512"
```

Comment:
why moving authentication section here?

Reply:
Test which use this as expected output failed every time before move this part, problem in output verification. Not sure why it happen only in this PR and not early.